data/remnant/remnant events.txt

```diff
@@ -176,8 +176,6 @@ event "remnant: cognizance 1 systems available"
 		remove attributes "inaccessible"
 	system Egeria
 		remove attributes "inaccessible"
-	system Porrima
```

Comment:
I think this should actually be a correction, replacing "Porrima" with "Postverta".